service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -84,6 +84,14 @@ class CaseData implements JsonSerializable
     #[Annotation\Required(false)]
     public ?string $searchPostcode = null;
 
+    #[Annotation\Required(false)]
+    #[Validator(Uuid::class)]
+    public ?string $sessionId = null;
+
+    #[Annotation\Required(false)]
+    #[Validator(Uuid::class)]
+    public ?string $notifyAuthToken = null;
```

Comment:
Notify has a particular meaning in gov, due to the GDS Notify service being called Notify, so best not use that here to avoid confusion. `NotificationsAuthToken` maybe?

Reply:
Might also be worth putting these inside a Yoti or CounterService model ($caseData->yoti->sessionId) to avoid confusion with our other integrations.